### ISSUE TITLE
`prefer-array-find` doesn't warn about `.at(0)`

### ISSUE THREAD

**Issue by EvgenyOrekhov** [sindresorhus/eslint-plugin-unicorn#2280]
`prefer-array-find` doesn't warn about `.at(0)`

This code should trigger a warning:

```js
const item = array.filter(x => isUnicorn(x)).at(0);
```
